src/Congo.WebApi/Controllers/CartController.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Congo.Contracts.Responses.Cart;
+using Congo.WebApi.Data.CartAccess;
+using Mapster;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Congo.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CartController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public CartController(IMediator mediator) => _mediator = mediator;
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<CartResponse>>> Get()
+        {
+            var carts = await _mediator.Send(new GetCartsQuery());
+            return Ok(carts.Adapt<IEnumerable<CartResponse>>());
```

Comment:
The service layer (our request handlers) should take care of mapping from our entities to response models

Reply:
No, that makes them less reusable. It's perfectly fine for the controller to do the mapping.